about.html

```diff
@@ -1,64 +1,104 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Sprint Challenge - About</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/about.css" />
+  </head>
+
+  <body>
+    <div class="container-about-page">
```

Comment:
You can do a header section nav here, but is the section doing anything?  Is there any styling applied to it?  If not, we can just omit it and have a header nav